model/sharing/indexer.go

```diff
@@ -293,7 +293,21 @@ func (s *sharingIndexer) UpdateDirDoc(olddoc, doc *vfs.DirDoc) error {
 	}
 	doc.SetRev(s.bulkRevs.Rev)
 	docs[0]["_rev"] = s.bulkRevs.Rev
-	docs[0]["_revisions"] = s.bulkRevs.Revisions
+	if olddoc != nil {
+		/*
+			XXX We cannot directly apply the revision chain as received by the remote
+			instance because it might create a CouchDB conflict if a rev does not
+			exist on the local instance. Therefore, we start from the last local rev
+			and apply all the higher revs received from the remote.
+		*/
```

Comment:
[nit] Can you use `//` for comments please? It would be more consistent with the rest of the code base.